app/views/profile_page.erb

```diff
@@ -0,0 +1,5 @@
+<div class="well">
+  <h3><%= @user.username %></h3>
```

Comment:
If you had used the `helper` keyword to include your `SessionHelper` you could access it in the template. Then you could declare a `current_user` method in the `SessionHelper` and change this to:

```erb
<h3><%= current_user.username %></h3>
<!-- etc -->
```

Then you wouldn't need to include the `@user` variable everywhere.